Library/Formula/glslang.rb

```diff
@@ -0,0 +1,21 @@
+class Glslang < Formula
+  desc "glslang is an OpenGL and OpenGL ES shader front end and validator."
+  homepage "https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/"
+  url "https://github.com/KhronosGroup/glslang/archive/3.0.tar.gz"
+  sha256 "91653d09a90440a0bc35aa490d0c44973501257577451d4c445b2df5e78d118c"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+      bin.install "StandAlone/glslangValidator"
+    end
+  end
+
+  test do
+    system "#{bin}/glslangValidator", "-v"
```

Comment:
Since the test fixtures are tiny and BSD licensed let's just inline them directly into `test do` with `(testpath/"something.extension").write <<-EOS.undent` and so on. You can add a small attribution if you'd like to.